modules/node_modules/@ncigdc/containers/CancerDistributionTable.js

```diff
@@ -99,7 +98,7 @@ const CancerDistributionTableComponent = compose(
           key={b.key}
           ssms={explore.ssms}
           filters={makeFilter([
-            { field: 'genes.gene_id', value: [geneId] },
+            { field: 'genes.symbol', value: [entityName] },
```

Comment:
I'm not sure that this change, it's the filter for fetching mutation counts for the gene in each project, not a link to the explore page and junjun's said there could be genes with multiple ids? Might change the counts.